javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,97 @@
+// voice assistant 
+
+let userName= "";
+const todolist=[];
+
+function getReply(command){
+if (command==="Hello my name is Benjamin" && userName==null){
+    return `You are not a person`
+}
+    if (command ==="Hello my name is Benjamin" && userName===""){
+        userName="Benjamin";
+        return `Nice to meet you ${userName}`;
+
+    } if(command ==="Hello my name is Benjamin" && userName!=""){
+        userName="";
+        return`This name was already mentioned`;
+ 
+    } if (command==="what is my name?"){
+    return`Your name is ${userName}`;
+
+}if (command ==="what is my name?" && userName.length===0){
+    return `Name is not mentioned`
+
+} if (command ==="Add fishing to my todo"){
+    todolist.push("fishing");
+    return `Fishing has been added to your todo list`
+
+}if (command ==="Add singing in the shower to my todo"){
+    todolist.push("singing in the shower")
+    return`Singing in the shower has been added to your todo list`
+
```

Comment:
This works! But, you could also check which action should be added to `todoList`, for example if a user tries with `getReply("Add eating cookies to my todo");` it would add `eating cookies` to the `todoList`. It can also be a function that would be called for every command `Add _____ to my todo`
Hint: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring